socs/agents/hwp_supervisor/agent.py

```diff
@@ -626,6 +659,18 @@ class PmxOff:
         """
         success: bool = True
 
+    @dataclass
+    class GripHWP:
```

Comment:
I think having a force option which will ignore ensure_grip_safety is helpful.
Obviously we should not use it usually but it might be helpful when we really need to grip it but the site is in unusual state.

Reply:
I can add this, but just want to make sure, are we sure this is something we really want? If the site is an unusual state it is still be possible to go in and manually run the command from the gripper agent, or even the windows software.

I'm not entirely sure we want to provide non-experts an easy way to ignore safety checks.

Reply:
That's fair point. I agree not to add it in hwp-supervisor.